tests/monitoring/metrics.go

```diff
@@ -141,19 +122,26 @@ func fetchPrometheusMetrics(virtClient kubecli.KubevirtClient) *libmonitoring.Qu
 	return metrics
 }
 
-func setupVM(virtClient kubecli.KubevirtClient) {
-	vm := createRunningVM(virtClient)
+func basicVMLifecycle(virtClient kubecli.KubevirtClient) {
+	By("Creating and running a VM")
+	vm := createAndRunVM(virtClient)
+
+	By("Waiting for the VM to be reported")
 	libmonitoring.WaitForMetricValue(virtClient, "kubevirt_number_of_vms", 1)
 
+	By("Waiting for the VM domainstats metrics to be reported")
+	libmonitoring.WaitForMetricValueWithLabelsToBe(virtClient, "kubevirt_vmi_filesystem_capacity_bytes", map[string]string{"namespace": vm.Namespace, "name": vm.Name}, 0, ">", 0)
+
 	By("Deleting the VirtualMachine")
 	err := virtClient.VirtualMachine(vm.Namespace).Delete(context.Background(), vm.Name, metav1.DeleteOptions{})
 	Expect(err).ToNot(HaveOccurred())
 
+	By("Waiting for the VM deletion to be reported")
 	libmonitoring.WaitForMetricValue(virtClient, "kubevirt_number_of_vms", -1)
 }
 
-func createRunningVM(virtClient kubecli.KubevirtClient) *v1.VirtualMachine {
-	vmi := libvmifact.NewGuestless(libvmi.WithNamespace(testsuite.GetTestNamespace(nil)))
+func createAndRunVM(virtClient kubecli.KubevirtClient) *v1.VirtualMachine {
+	vmi := libvmifact.NewFedora(libvmi.WithNamespace(testsuite.GetTestNamespace(nil)))
```

Comment:
@machadovilaca Why did you change it to a bootable guest?

Reply:
Now testing the domainstats metrics, a bootable guest is need to ensure they are collected and with all the information we expose